tree/dataframe/src/RDFUtils.cxx

```diff
@@ -289,7 +289,9 @@ std::vector<std::string> ReplaceDotWithUnderscore(const std::vector<std::string>
 void InterpreterDeclare(const std::string &code)
 {
    if (!gInterpreter->Declare(code.c_str())) {
-      const auto msg = "\nAn error occurred while jitting. The lines above might indicate the cause of the crash\n";
+      const auto msg =
+         "\nRDataFrame: An error occurred while jitting. The lines above might indicate the cause of the crash\n All "
```

Comment:
Is `jitting` something all users understand? Maybe one could also say compiling?
Clearly not important for functioning of this feature, though...

Reply:
I'll rephrase to "An error occurred during just-in-time compilation"